content/en/docs/components/katib/experiment.md

```diff
@@ -594,73 +621,87 @@ Katib supports the following algorithm settings:
         <td>stem_multiplier</td>
         <td>int</td>
         <td>3</td>
-        <td>Multiplier for initial channels. It is used in first stem cell.</td>
+        <td>Multiplier for initial channels. It is used in the first stem cell.</td>
       </tr>
     </tbody>
   </table>
 </div>
 
-For more information, see:
+For more information, check:
+
+- Documentation in the Katib repository on the
+  [Differentiable Architecture Search](https://github.com/kubeflow/katib/tree/master/pkg/suggestion/v1beta1/nas/darts).
 
-* Information in the Katib repository on [Differentiable Architecture Search](https://github.com/kubeflow/katib/tree/master/pkg/suggestion/v1alpha3/nas/darts).
-* As a DARTS example, see the YAML file for the
-[darts-example-gpu](https://github.com/kubeflow/katib/blob/master/examples/v1alpha3/nas/darts-example-gpu.yaml).
+- As a DARTS example, check the YAML file for the
+  [darts-example-gpu](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/nas/darts-example-gpu.yaml).
 
 <a id="metrics-collector"></a>
+
 ### Metrics collector
 
 In the `metricsCollectorSpec` section of the YAML configuration file, you can
-define how Katib should collect the metrics from each trial, such as the 
-accuracy and loss metrics. See the [`MetricsCollectorSpec` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1alpha3/common_types.go#L81-L150)
+define how Katib should collect the metrics from each trial, such as the
+accuracy and loss metrics.
+Refer to the
+[`MetricsCollectorSpec` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1beta1/common_types.go#L155-L225)
 
 Your training code can record the metrics into `stdout` or into arbitrary output
-files. Katib collects the metrics using a *sidecar* container. A sidecar is
+files. Katib collects the metrics using a _sidecar_ container. A sidecar is
 a utility container that supports the main container in the Kubernetes Pod.
```

Comment:
Check if a K8s "pod" is capitalized

Reply:
I think so: https://kubernetes.io/docs/concepts/workloads/pods/